dashboard/app/models/user.rb

```diff
@@ -286,12 +287,12 @@ def admins_must_be_teachers_without_followeds
 
   def email
     return read_attribute(:email) unless migrated?
-    primary_authentication_option.try(:email)
+    primary_authentication_option.try(:email) || ''
   end
 
   def hashed_email
     return read_attribute(:hashed_email) unless migrated?
-    primary_authentication_option.try(:hashed_email)
+    primary_authentication_option.try(:hashed_email) || ''
```

Comment:
The `email` and `hashed_email` fields on `AuthenticationOption` are `NOT NULL DEFAULT ''`, but this method of retrieving them could still return `nil` if no options are present.  It seemed to simplify logic elsewhere if we normalized these proxy methods to never return `nil`.